src/utils/object_helpers.js

```diff
@@ -0,0 +1,18 @@
+/* Useful sky object related functions */
+var object_helpers = {
```

Comment:
var works fine, but it's been superceded and therefore probably better replaced by either const or let (const in this case).